spec/factories/quota_factory.rb

```diff
@@ -67,6 +101,8 @@
     monetary_unit_code              { Forgery(:basic).text(exactly: 3) }
     measurement_unit_code           { Forgery(:basic).text(exactly: 3) }
     measurement_unit_qualifier_code { generate(:measurement_unit_qualifier_code) }
+    validity_start_date             { Date.current.ago(4.years) }
```

Comment:
This is a timestamp field - `4.years.ago` might make more sense but no strong feelings